src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -16,6 +16,7 @@
 src/arcade/Documentation/UnifiedBuild/Foundational-Concepts.md
 src/arcade/src/Microsoft.DotNet.Build.Tasks.Installers/src/BuildFPMToolPreReqs.cs|json
 src/arcade/src/Microsoft.DotNet.Build.Tasks.Installers/build/rpm_templates/copyright|cecill-c
+src/arcade/src/Microsoft.DotNet.XUnitAssert/src/README.md|free-unknown
```

Comment:
this is actually Apache 2.0 according to https://github.com/dotnet/arcade/blob/c5e5051496efbcc9046b25628666377d6b97a670/THIRD-PARTY-NOTICES.TXT#L13-L31

Reply:
It's detecting both `free-unknown` and `apache-2.0` but the `free-unknown` is a false positive. This is for the following line: https://github.com/dotnet/arcade/blob/c5e5051496efbcc9046b25628666377d6b97a670/src/Microsoft.DotNet.XUnitAssert/src/README.md?plain=1#L102